flexmeasures/ui/templates/crud/user.html

```diff
@@ -31,23 +31,6 @@ <h2> Account overview for {{ user.username }} </h2>
                 </form>
             </div>
             <div class="col-sm-10">
```

Comment:
Do we need to keep this div for spacing?

Reply:
Yes, to define the 12 columns of Bootstrap.